apps/eas-expo-go/eas.json

```diff
@@ -91,6 +71,40 @@
         "simulator": true,
         "buildConfiguration": "Release"
       }
+    },
+    "release-client": {
+      "extends": "versioned-client",
```

Comment:
do we need to set `"autoIncrement": true` here? [given that the default is `false`](https://docs.expo.dev/build-reference/eas-json/#autoincrement)

Reply:
yes, I planned to add that after testing the full flow including submitting it to the store, there are also a few other change I mentioned in todos in a code
- add `--auto-submit`
- switch track to production for android